src/plots/plot.ts

```diff
@@ -249,8 +253,8 @@ export class Plot extends Component {
    * Updates the extents associated with each attribute, then autodomains all scales the Plot uses.
    */
   protected _updateExtents() {
-    this._attrBindings.forEach((attr) => this._updateExtentsForAttr(attr));
-    this._propertyExtents.forEach((property) => this._updateExtentsForProperty(property));
+    this._attrBindings.each((_, attr) => this._updateExtentsForAttr(attr));
+    this._propertyExtents.each((_, property) => this._updateExtentsForProperty(property));
```

Comment:
in these cases, i've seen people do `...each((_binding, property) => ...` to get the best of both worlds